src/buza/models.py

```diff
@@ -31,3 +48,26 @@ class User(AbstractUser):
 
     def __str__(self) -> str:
         return str(self.username)
+
+
+class Question(TimestampedModel, models.Model):
+
+    author = models.ForeignKey(User, on_delete=models.PROTECT)
+
+    title = _CharField()
+    body = models.TextField(blank=True)
+
+    def __str__(self) -> str:
+        return f'By {self.author}: {self.title}'
+
+
+class Answer(TimestampedModel, models.Model):
+
+    author = models.ForeignKey(User, on_delete=models.PROTECT)
```

Comment:
from my understanding we protect the author to prevent a whole bunch of import questions being deleted if admin accidentally deletes a user. But it's okay to cascade the answers because users can delete a questions.

Reply:
Right.

These just specify the database-level constraints for the RDBMS, though, so it's not the whole picture of the policy around deletions: we'll probably get more nuanced about that later.